install_bootstrap

```diff
@@ -426,11 +436,11 @@ while [[ "${run_dockstore_launcher^^}" != 'Y' &&  "${run_dockstore_launcher^^}"
 "BITBUCKET_CLIENT_SECRET":"${bitbucket_client_secret}",
 "COMPOSE_SETUP_VERSION":"${compose_setup_version}",
 "DOMAIN_NAME":"${domain_name}",
-"ELASTICSEARCH_DOMAIN"="${elasticsearch_domain}",
-"ELASTICSEARCH_PASSWORD"="${elasticsearch_password}",
-"ELASTICSEARCH_PORT"="${elasticsearch_port}",
-"ELASTICSEARCH_PROTOCOL"="${elasticsearch_protocol}",
-"ELASTICSEARCH_USER"="${elasticsearch_user}",
+"ELASTICSEARCH_DOMAIN":"${elasticsearch_domain}",
```

Comment:
> Did you test that the digest gets overwritten when subsequent builds of a branch are made? Should have asked that in your other PR, but that will need to work for our develop branch deploys.

It will pull the latest commit on a branch and check if its been uploaded to S3.

Reply:
> Whoops, how did I do that and I'm surprised it didn't break things.

None of the tests check the interactive part of install_bootstrap, everything uses --script with a premade mustache json.

Reply:
> > Whoops, how did I do that and I'm surprised it didn't break things.
> 
> None of the tests check the interactive part of install_bootstrap, everything uses --script with a premade mustache json.

I meant I'm surprised the CF template that relies on this didn't fail, but now that I think about it, the deployer writes out its own JSON

Reply:
> > Did you test that the digest gets overwritten when subsequent builds of a branch are made? Should have asked that in your other PR, but that will need to work for our develop branch deploys.
> 
> It will pull the latest commit on a branch and check if its been uploaded to S3.

Ah, I see, you upload/download to `<branch>-<commit>`; so builds' digests won't overwrite each other.